src/script/entity/User/User.ts

```diff
@@ -54,6 +54,7 @@ export class User {
   public readonly connection: ko.Observable<ConnectionEntity>;
   /** does not include current client/device */
   public readonly devices: ko.ObservableArray<ClientEntity>;
+  public localClient: ko.Observable<ClientEntity> | undefined;
```

Comment:
It doesn't need to be observable. We're assigning it only once on app init and never update the reference.